src/ert/_c_wrappers/enkf/_deprecation_migration_suggester.py

```diff
@@ -61,8 +62,12 @@ def _add_deprecated_keywords_to_parser(self):
     def suggest_migrations(self, filename: str):
         suggestions = []
         content = self._parser.parse(
-            filename, pre_defined_kw_map=self._pre_defines, validate=False
+            filename, pre_defined_kw_map=self._pre_defines, validate=True
```

Comment:
`validate=True` means that the below raise is done by `self._parser.parse` so it is redundant.

Also, this is too soon to do the validation because we do not yet have the site config parsed. Therefore this will give false positives. Instead we need to not validate at this point. Check `content.isValid()` and then if that is false, return empty list.